### PR TITLE
geo location action docs

### PR DESCRIPTION
**Pull request by simcha-waintrob**

### CODE REVIEW

File: actions/geo-location.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+You can adjust the accuracy level of the location.\
+It is also possible to wait a certain time until the desired level of accuracy is obtained
+
+### Parameters
+
+
+#### Accuracy
+The desired level of accuracy of the location.\
+There are three levels of accuracy:
+* HIGH (Accuracy of meters)
```
[the remark this answers is not in the document]
Comment (Chasky-H) [reply actions/geo-location.md:15]: See if this definition will hold up for android and web as well

File: actions/geo-location.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+* MEDIUM (Accuracy of hundreds of meters)
+* LOW (Accuracy of a kilometer)
+
+
+
+
+#### WaitTime
```
Comment (Chasky-H) [inline actions/geo-location.md:22]: MaxWaitTime
or
MaxCaptureTime

File: actions/geo-location.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+A number in milliseconds between 0 - 9999 ms.\
+To wait for a response from the GPS services.\
+If the location is reached sooner than the indicated time, the result will return immediately.\
+If a location is not received within the indicated time, the action will fail.
```
Comment (Chasky-H) [inline actions/geo-location.md:26]: The action can return whatever accuracy we did manage to capture in that time.
Is there any way we can return the accuracy to the user (not sure this is available on all platforms)

File: actions/geo-location.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  "Latitude": "-120.740135"
+}
+```
+
+## Usages
+we can capture the current location from  CPI Node like the following example:
+
+#### Example 
+```typescript
+const res = await pepperi.client.captureGeoLocation('HIGH', 3000);
```
Comment (Chasky-H) [inline actions/geo-location.md:66]: parameters should always be an object so that we can add stuff later without breaking the API

**Review by Chasky-H — CHANGES_REQUESTED** [review]

File: actions/geo-location.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (simcha-waintrob) [reply actions/geo-location.md:15]: The web has regular and high.
Android has also 3 levels.

File: actions/geo-location.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
+# Capture Geo Lcation
+Capture current user's geo location.
```
Comment (Chasky-H) [inline actions/geo-location.md:2]: user's current geo location

File: actions/geo-location.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+```
+
+## Usages
+we can capture the current location from  CPI Node like the following example:
+
+#### Example 
+```typescript
+const res = await pepperi.client.captureGeoLocation({accuracy: 'High', maxWaitTime: 3000});
+
+if (res.Success) {
```
Comment (Chasky-H) [inline actions/geo-location.md:70]: res.success

Comment (Chasky-H) [reply actions/geo-location.md:70]: all client API is cpi-node should be camelCase
all Client Action API should be PascalCase

**Review by Chasky-H — APPROVED** [review]

**Comment by Chasky-H** [comment]
@simcha-waintrob please merge master into your branch

**Comment by simcha-waintrob** [comment]
> @simcha-waintrob please merge master into your branch

done

**Review by Chasky-H — APPROVED** [review]

**Pull request merged**